primitive/nillables.go

```diff
@@ -25,3 +25,27 @@ type NillableInt64 struct {
 type NillableConsistencyLevel struct {
 	Value ConsistencyLevel
 }
+
+func CloneNillableInt32(n *NillableInt32) *NillableInt32 {
```

Comment:
The more it goes the more I think these "Nillable" types could be avoided... But we can think about it in another PR. Here, I'd just suggest to turn these into methods on the receiver type (and taking care of nil receivers).